react-app/src/components/RomcalVersion.tsx

```diff
@@ -9,10 +9,17 @@ const VersionContainer = styled('div')`
   font-size: 0.8em;
   color: #7f7f7f;
   font-weight: 100;
+  display: flex;
+  justify-content: center;
+  gap: 10px;
 `;
 
-const RomcalVersion = observer(() => {
-  return <VersionContainer>romcal v.{Romcal.getVersion()}</VersionContainer>;
-});
+const RomcalVersion = observer(() => (
+  <VersionContainer>
+    <span>romcal v.{Romcal.getVersion()}</span>
+    <span> – </span>
+    <span>app commit {COMMIT_HASH}</span>
```

Comment:
I suggest to change the `app` to something else here. As I understand it, the version displayed is the romcal _library_ version, while the hash of the romcal _app_.

Moreover, I don’t think that the dot after `v` (i.e. `v.`). Usually, we format the version like `v3.0.0-dev.67` or `3.0.0-dev.67`. I’ve never seen `v.3.0.0-dev.67`.

```suggestion
    <span>romcal library v{Romcal.getVersion()}</span>
    <span> – </span>
    <span>romcal app commit {COMMIT_HASH}</span>
```

We might also want to simplify this. In the suggested code above, I use the `romcal` word twice. Something like this, however, it might need some tweaks (esp from the UI point of view):

```js
romcal version:
- library: v3.0.0-dev.67
- app commit: f521712
```

This has, however, a subtle issue: the romcal app is not (yet) versioned (we show a commit hash here), thus `romcal version` a bit incorrect.

Reply:
Thank you for your review.

Here, it's purely technical information displayed discreetly at the bottom. So I believe it's not a big deal if there's a bit of repetition here (the important thing is to be precise).

![Screenshot 2024-06-21 at 13 30 10](https://github.com/romcal/romcal-examples/assets/1045997/d458b5d1-438b-4f33-a985-663cacc2a51a)